lib/session_encryptor.rb

```diff
@@ -23,7 +23,6 @@ class SensitiveValueError < StandardError; end
   # personal keys are generated and stored in the session between requests, but are used
   # to decrypt PII bundles, so we treat them similarly to the PII itself.
   SENSITIVE_PATHS = [
-    ['warden.user.user.session', 'idv/doc_auth'],
```

Comment:
This should be find since we are no longer writing anything to this path, much less PII.

Reply:
The comment above this mentions idv/doc_auth and should probably be updated